ClientApp/viking-vault/src/UserDashboard/TransferFunds/TransferFundsPage.tsx

```diff
@@ -0,0 +1,340 @@
+            transferedAmount: e.target.value
+          });
+        }
+    }
+
+    handleChangedCardNumber = (e : any) => {
+        this.setState({
+            cardNumber: e.target.value
+        })
+    }
+
+    handleChangedTransferDetails = (e : any) => {
+        this.setState({
+            transferDetails: e.target.value
+        })
+    }
+
+    private setCurrency = (e: any) => {
+      this.setState({
+        currency: e.target.value
+      });
+      this.getTotalBalance();
+    }
+
+    handleChangeRequestTransferState = () => {
+      this.setState((oldState: ITransferFundsState) => ({
+        requestTransfer: !oldState.requestTransfer
+      }));
+        
+    }
+
+    handleIsButtonDisabled = () => {
+        this.setState((oldState: ITransferFundsState) => ({
+          isButtonDisabled: !oldState.isButtonDisabled
+        }));
+    }
+
+    getTotalBalance = () => {
+      let token = sessionStorage.getItem('Authentication-Token');
+        if(token != null) {
+            fetch(constants.baseUrl+"bankAccount", {
+                method: "GET",
+                headers: {
+                  'Accept': 'application/json',
+                  'Content-Type': 'application/json',
+                  'x-access-token': token.toString()
+                }
+            })
+            .then(response => response.json())
+            .then(result => {
+              this.setState({
+                totalBalance: result[currencyMap[this.state.currency]].balance
+              });
+            });
+        }
+    }
+
+    autoFillTransferRequest = (transferData: any) => {
+      this.setState({
+        transferedAmount: transferData.amount,
+        currency: transferData.currency,
+        cardNumber: transferData.cardNumberRequester,
+        transferDetails: "Responding to your request to transfer money.",
+        requestTransfer: false,
+        isTransferRequested: true,
+        requestId: transferData.id
+      })
+    }
+
+    changeReloading = (reload: boolean) =>
+    {
+      this.setState({
+        reloadRequestsData: reload
+      })
+    }
+
+    private getTransferDataFromUI = ():ITransferDataDTO => {
+        return {
+          amount: this.state.transferedAmount,
+          cardNumberReciever: this.state.cardNumber,
+          transferDetails: this.state.transferDetails,
+          currency: this.state.currency
+        };  
+    }
+
+    private isValidAmount = (data: ITransferDataDTO) => {
+      if(data.amount == 0){
+          this.setState({
+            openModal: true,
+            modalMessage: "Insert a proper amount to be sent!"
+          })
+
+          return false;
+      }
+
+      return true;
+    }
+
+    private getDataForTransfer = () => {
+          return {
+              ...this.getTransferDataFromUI(),
+              isTransferRequested: this.state.isTransferRequested,
+              requestId: this.state.requestId
+          }
+    }
+
+    transferMoney = () => {
+      
```

Comment:
please remove this empty line. Also, the formatting is a bit weird for this method. Can you check it out?

Reply:
What do you mean by formatting?

Reply:
Nevermind, solved it.